src/app/shell/personal-cabinet/applications/application-card/application-card.component.html

```diff
@@ -1,44 +1,52 @@
 <mat-card *ngIf="application" fxLayout='row' fxLayoutAlign='space-evenly center'
   [ngClass]="(application.status === applicationStatus.pending) ? 'card card-new' : 'card'">
-  <mat-card-content class="card-block" fxLayout='row' fxLayoutAlign='start center'>
+  <mat-card-content class="card-block" fxLayout='row' fxLayoutAlign='start center' id="first-column">
     <div class="avatar" *ngIf="userRole === Role.provider">
       <img mat-card-avatar #img src="assets/icons/child-{{application.child.gender}}.png" alt="child-photo"
         (mouseover)="onInfoShow(img) " (mouseleave)="onInfoHide(img)">
     </div>
-    <div fxLayout='column' fxLayoutAlign='start start'>
-      <h5>{{ application.child.firstName }} {{ application.child.lastName }} {{ application.child.middleName }}</h5>
-      <p class="card-text">Дата народження: {{ application.child.dateOfBirth | date: 'shortDate'}}</p>
-      <p class="card-text">Дата подання заявки: {{ application.creationTime | date: 'shortDate' }}</p>
+    <div class="first-column" fxLayout='column' fxLayoutAlign='start start' >
```

Comment:
it is a little bit confusing to see first-column class and id set to different elements. could you please check if we still need this?